packages/desktop-client/src/components/manager/BudgetList.tsx

```diff
@@ -467,7 +493,16 @@ export function BudgetList({ showHeader = true, quickSwitchMode = false }) {
         files={files}
         quickSwitchMode={quickSwitchMode}
         onSelect={onSelect}
-        onDelete={file => dispatch(pushModal('delete-budget', { file }))}
+        onDelete={(file: File) =>
+          dispatch(pushModal('delete-budget', { file }))
+        }
+        onDuplicate={(file: File) => {
+          if (file && 'id' in file) {
+            dispatch(pushModal('duplicate-budget', { file, managePage: true }));
+          } else {
+            console.error('Attempted to duplicate an invalid file:', file);
```

Comment:
```suggestion
            console.error('Attempted to duplicate a cloud file - only local files are supported. Cloud file:', file);
```

Reply:
Fixed in commit [882c14d](https://github.com/actualbudget/actual/pull/3847/commits/882c14d58db1bd059bf3daa5c98379f680d6870a)